commit 1e12605eac293b12809ca2933622acc54515c237
Author: Nils Philippsen <nils@redhat.com>
Date:   Mon Dec 28 17:00:25 2009 +0100

    patch: lockdir
    
    Squashed commit of the following:
    
    commit 7d26ebc2c0c0c8a8e3254834a700951b49648eeb
    Author: Nils Philippsen <nils@redhat.com>
    Date:   Mon Dec 28 16:58:36 2009 +0100
    
        improve install-lockpath target
    
        make robust against problems with setting group on sane lock dir

diff --git a/backend/Makefile.am b/backend/Makefile.am
index 5165423..3c22cd4 100644
--- a/backend/Makefile.am
+++ b/backend/Makefile.am
@@ -137,7 +137,8 @@ install-firmware-path:
 	  done
 
 install-lockpath:
-	$(mkinstalldirs) -m 775 -g $(LOCKPATH_GROUP) $(DESTDIR)$(locksanedir)
+	$(mkinstalldirs) -m 775 -g $(LOCKPATH_GROUP) $(DESTDIR)$(locksanedir) || \
+		$(mkinstalldirs) -m 775 $(DESTDIR)$(locksanedir)
 
 uninstall-hook:
 	rm -rf $(DESTDIR)$(libdir)/sane $(DESTDIR)$(configdir) $(DESTDIR)$(locksanedir)
